x-pack/plugins/cross_cluster_replication/public/app/sections/home/auto_follow_pattern_list/components/detail_panel/detail_panel.js

```diff
@@ -0,0 +1,303 @@
+
+import {
+  EuiButton,
+  EuiDescriptionList,
+  EuiDescriptionListDescription,
+  EuiDescriptionListTitle,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiFlyoutHeader,
+  EuiIcon,
+  EuiLoadingSpinner,
+  EuiSpacer,
+  EuiText,
+  EuiTextColor,
+  EuiTitle,
+} from '@elastic/eui';
+
+import {
+  FollowerIndicesPreview,
+  AutoFollowPatternDeleteProvider,
+} from '../../../../../components';
+
+import { getPrefixSuffixFromFollowPattern } from '../../../../../services/auto_follow_pattern';
+import routing from '../../../../../services/routing';
+
+export class DetailPanelUi extends Component {
+  static propTypes = {
+    isDetailPanelOpen: PropTypes.bool.isRequired,
+    isLoading: PropTypes.bool,
+    autoFollowPattern: PropTypes.object,
+    autoFollowPatternName: PropTypes.string,
+    closeDetailPanel: PropTypes.func.isRequired,
+    editAutoFollowPattern: PropTypes.func.isRequired,
+  }
+
+  renderAutoFollowPattern() {
+    const {
+      autoFollowPattern: {
+        followIndexPattern,
+        remoteCluster,
+        leaderIndexPatterns,
+      },
+    } = this.props;
+
+    const {
+      followIndexPatternPrefix,
+      followIndexPatternSuffix,
+    } = getPrefixSuffixFromFollowPattern(followIndexPattern);
+
+    return (
+      <Fragment>
+        <EuiFlyoutBody>
+          <EuiTitle size="s">
+            <h3>
+              <FormattedMessage
+                id="xpack.crossClusterReplication.autoFollowPatternDetailPanel.statusTitle"
+                defaultMessage="Settings"
+              />
+            </h3>
+          </EuiTitle>
+
+          <EuiSpacer size="s" />
+
+          <EuiDescriptionList>
+            <EuiFlexGroup>
+              <EuiFlexItem>
+                <EuiDescriptionListTitle>
+                  <EuiTitle size="xs">
+                    <FormattedMessage
+                      id="xpack.crossClusterReplication.autoFollowPatternDetailPanel.remoteClusterLabel"
+                      defaultMessage="Remote cluster"
+                    />
+                  </EuiTitle>
+                </EuiDescriptionListTitle>
+
+                <EuiDescriptionListDescription>
+                  {remoteCluster}
+                </EuiDescriptionListDescription>
+              </EuiFlexItem>
+
+              <EuiFlexItem>
+                <EuiDescriptionListTitle>
+                  <EuiTitle size="xs">
+                    <FormattedMessage
+                      id="xpack.crossClusterReplication.autoFollowPatternDetailPanel.leaderPatternsLabel"
+                      defaultMessage="Leader patterns"
+                    />
+                  </EuiTitle>
+                </EuiDescriptionListTitle>
+
+                <EuiDescriptionListDescription>
+                  {leaderIndexPatterns.join(', ')}
+                </EuiDescriptionListDescription>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+
+            <EuiSpacer size="s" />
+
+            <EuiFlexGroup>
```

Comment:
I just found a visual bug. When there is no `prefix` or `suffix` defined, could we render `No suffix` and `No prefix` inside of `<em>` tags? Right now they appear blank:

![image](https://user-images.githubusercontent.com/1238659/50114255-f5de1400-01f8-11e9-968b-45c0efb80db8.png)